### ISSUE TITLE
Charts display despite no data

### ISSUE THREAD

**Issue by brentryanjohnson** [18F/doi-extractives-data#2709]
## Expected Behavior

- If no data exists for 10-year range of a commodity, it won't display a chart

## Actual Behavior

- Chart displays for [Hydroelectric Energy under Kentucky wage and salary jobs](https://revenuedata.doi.gov/explore/KY/#employment) despite no data.

![hydro chart shows up despite containing no data](https://user-images.githubusercontent.com/32855580/37547349-5c77761a-292e-11e8-90a8-d43441e4ecc1.png)

**Comment by brentryanjohnson** [comment]
[Same is happening in Ohio for hydroelectric](https://revenuedata.doi.gov/explore/OH/#employment).

**Comment by brentryanjohnson** [comment]
Same is happening for [Washington coal](https://revenuedata.doi.gov/explore/WA/#all-production).

![Washington coal chart with no data](https://user-images.githubusercontent.com/32855580/37548893-7673bea2-2938-11e8-85ce-2417c9301fe5.png)

**Comment by jennmalcolm** [comment]
Next step:  @brentryanjohnson review raw data to see if you can determine why this is happening.

**Comment by brentryanjohnson** [comment]
Looking into some patterns that may be causing this issue, I've observed the following:

## Kentucky hydroelectric jobs

We have a data value for hydroelectric jobs only for 2006. Our charted values start at 2007.

```Hydroelectric:
    '2006':
      count: 54
      total: 1779202
      percent: 0
```
Hypothesis: Even though the data is out of the year range, the chart may be activating based on the presence of any data value. Next step: remove the value and see if the chart disappears.

**Update**: I removed the hydroelectric data in [`state_jobs_by_commodity.yml`](https://github.com/18F/doi-extractives-data/blob/dev/_data/state_jobs_by_commodity.yml), and the chart remained. So something else is going on.

## Ohio hydroelectric jobs

Ohio also has only one year for which we have data in this category.

```  Hydroelectric:
    '2006':
      count: 1936
      total: 5314572
      percent: 0.04
```
It's a similar situation to Kentucky, wherein we have one data value for a year we are not charting.

## Washington coal production

All years for which we have coal production have a value of `null`. This is very likely due to withheld data.
This can be confirmed by reviewing the raw data.

```
WA:
  products:
    Coal (tons):
      name: Coal
      units: tons
      volume:
        '2014':
          volume: null
          percent: null
          rank: null
        '2015':
          volume: null
          percent: null
          rank: null
        '2016':
          volume: null
          percent: null
          rank: null
```

**Comment by jennmalcolm** [comment]
Next step: @brentryanjohnson pair with @jeremiak to identify issue.

**Comment by jennmalcolm** [comment]
Fixed by #2759
